### ISSUE TITLE
`removeCookie` sends HTTP-Header that is ignored by the Browser

### ISSUE THREAD

**Issue by cassinius** [elysiajs/elysia#255]
When I use `removeCookie(<name>)`, the headers sent are:
```
'Set-Cookie': <cookie>=; Expires=Thu, 01 Jan 1970 00:00:00 GMT
```
which is ignored by current Chrome & Firefoxes.

However, if I send the http-header manually in such a way:
```
'Set-Cookie': '<cookie>=; Max-Age=0; Path=/; HttpOnly; SameSite=Lax; Domain=localhost',
```
the cookie is correctly removed.

**Comment by itohatweb** [comment]
There seems to be a similar issue with `cookie.remove()`

**Comment by SaltyAom** [comment]
Fixed in #281
